### DISCUSSION TITLE
example multi sort

### DISCUSSION THREAD

**Discussion by misaku** [TanStack/table#4811] [General]
i try to code multisort but i cant, i search about and i cant finded.
=(

**Comment by victorkvarghese** [comment]
Refer: https://tanstack.com/table/v8/docs/api/features/sorting?from=reactTableV7&original=https://react-table-v7.tanstack.com/docs/api/useSortBy#enablemultisort

you can enable multi sort using enablemultisort.

I think in windows you need to use windows key and in mac os need to use command key and click on multiple column headers to trigger multi sort

**Comment by misaku** [comment]
a understand now i need press shift to work multisort =)

**Comment by ejabu** [comment]
actually you can configure it so you don't need to press shift as seen below


![image](https://github.com/user-attachments/assets/7e1e9c75-9921-4a99-8a6d-cb78efe8c3de)

as seen here

https://tanstack.com/table/v8/docs/framework/react/examples/sorting
